core/templates/services/playthrough-issues-backend-api.service.ts

```diff
@@ -35,7 +35,7 @@ import { UrlInterpolationService } from
 
 @Injectable({ providedIn: 'root' })
 export class PlaythroughIssuesBackendApiService {
-  private cachedIssues = null;
+  private cachedIssues: PlaythroughIssue[] = [];
```

Comment:
Previously we resolved with `null`, now we resolve with `[]`, is that fine?

Reply:
yes, I have made changes in the file which reflect resolving with `[ ]`. `null` was being used to indicate that the cachedIssues have not been fetched. an empty list serves the same purpose.